tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -265,6 +295,17 @@ <h3>No profile data was found.</h3>
         type: String,
         value: "/trace_viewer_index.html",
       },
+      profilerServiceAddress: {
```

Comment:
It looks like these are not a public API of this component. Please prefix these props with underscore the privateness.

Reply:
But it is updated by html similar to other public properties like selectedToolIndex, sellectedHostIndex, etc.

Reply:
Either those props are used by parent or they are mistakes too :)

Reply:
But it can be modified by html similar to other public properties.